[ONNX] Bump onnx-script version with imported module renaming

titaiwangms opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #99927
* __->__ #99926

Avoid potential blocking after https://github.com/microsoft/onnxscript/pull/659

titaiwangms committed:
[ONNX] Bunp onnx-script version with imported module renaming

[ghstack-poisoned]

justinchuby commented on .ci/docker/common/install_onnx.sh:
```diff
@@ -27,7 +27,7 @@ pip_install \
 
 # TODO: change this when onnx-script is on testPypi
-pip_install "onnx-script@git+https://github.com/microsoft/onnxscript@1e8d764a9be04323d7171e4d5f511332790cb809"
+pip_install "onnx-script@git+https://github.com/microsoft/onnxscript@99a665546d374af872cf136137b5d8334389b78d"
```
```suggestion
pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@99a665546d374af872cf136137b5d8334389b78d"
```

it was renamed

justinchuby commented:
I Will wait for you to merge this before #99284

titaiwangms commented:
> I will wait for you to merge this before #99284

Sorry, I will try merge it tonight!

titaiwangms committed:
Update on "[ONNX] Bump onnx-script version with imported module renaming"


Avoid potential blocking after https://github.com/microsoft/onnxscript/pull/659

[ghstack-poisoned]

titaiwangms commented:
@pytorchbot merge

titaiwangms commented:
@pytorchbot merge
